### ISSUE TITLE
Stock splits cause incorrect tallying

### ISSUE THREAD

**Issue by anthonyk1225** [anthonyk1225/robinhood-to-xlsx#21]
If holding options or shares of a company when a stock split or reverse stock split happens, the profit/loss on sheet 2 for `options.xlsx` or `orders.xlsx` won't be accurate for those companies.

**Comment by kfchou** [comment]
Was just about to submit an issue on this. Glad you're already working on it!

**Comment by anthonyk1225** [comment]
Yep! Doing a bit of recon on it so may take a little bit longer. I'll have a solution eventually :D

**Comment by kfchou** [comment]
fyi, another scenario in addition to splits or reverse splits is when a company itself splits into two separate companies, or participate in mergers. For example, $SIR split into $OPI and $ILPT in 2018.
